src/services/jwt.services.js

```diff
@@ -0,0 +1,53 @@
+const jwt = require('jsonwebtoken');
+const { userDto } = require('../dto/user.dto');
+const { ApiError } = require('../exeptions/auth.error');
+
+const sign = (payload, key, expiresIn) => jwt.sign(payload, key, { expiresIn });
+
+const verify = (token, key) => {
+  try {
+    return jwt.verify(token, key);
+  } catch (error) {
+    if (error.name === 'TokenExpiredError') {
+      throw ApiError.unauthorized('The token has expired');
+    } else if (error.name === 'JsonWebTokenError') {
+      throw ApiError.unauthorized('The token is invalid');
+    } else {
+      throw ApiError.unauthorized('Token verification failed');
+    }
+  }
+};
+
+const generateAccessToken = (payload) =>
+  sign(payload, process.env.JW_ACCESS_KEY, '10m');
+
+const generateRefreshToken = (payload) =>
+  sign(payload, process.env.JW_REFRESH_KEY, '30d');
```

Comment:
Ensure that `process.env.JW_ACCESS_KEY` and `process.env.JW_REFRESH_KEY` are correctly set in your environment variables and are distinct from each other to avoid security issues.